npm/src/upload/index.ts

```diff
@@ -74,6 +82,9 @@ export class FileUploader {
   }
 
   initialiseFormListeners(): void {
+    const isJudgmentUser: boolean =
```

Comment:
Does this still work if no `judgment_user` on the token?

Reply:
Yes, it should work because if `judgment_user` is not there, it would return `undefined` and since that is not `true`, it will just make the value `false`